packages/@romejs/diagnostics/descriptions.ts

```diff
@@ -458,6 +458,10 @@ export const descriptions = createMessages(
             ],
           };
       },
+      RESTRICTED_GLOBALS: (globalName) => ({
+        category: 'lint/restrictedGlobals',
+        message: `don't use global variable ${globalName}, use local variable instead`,
```

Comment:
This looks great! Just a suggestion on the diagnostic message:

```suggestion
        message: markup`<emphasis>${globalName}</emphasis> isn't defined locally but is a valid global variable`,
```

Reply:
this new diagnostic message is a little bit confusing for me. We don't inform the user the global variable use is disallowed. I try to change the diagnostic message to: `the use of the existing global variable <emphasis>${globalName}</emphasis> is not allowed. Use local parameter instead.`. Say to me what do you think about this one.

Reply:
Few nits: It's not capitalized. "Parameter" isn't broad enough, a parameter is specifically a function parameter, we should call it a binding/variable.

Reply:
thanks for the review, I push some corrections.